admin/modules/user/groups.php

```diff
@@ -673,6 +673,23 @@
 		{
 			$errors[] = $lang->error_missing_title;
 		}
+		
+		if ($mybb->input['title'])
```

Comment:
Conditionals and round brackets should not be separated with a space. Applies to the whole PR.